utilities/policy_utilities.mak

```diff
@@ -21,7 +21,11 @@ endif
 #GOOGLE_INCLUDE=/usr/local/include/google
 #endif
 
+# Allows user to over-ride libs path externally depending on machine's install
+ifndef LOCAL_LIB
```

Comment:
Fixes inconsistent use of `LOCAL_LIB`. On sgx machine, with this fix, if you do:

`LOCAL_LIB=/usr/local/lib64 make -f policy_utilities.mak`, things will work correctly.